magento/magento-cloud-patches
# MAGECLOUD-4649: Merge PageBuilder Security Patches to m-cloud-patches

### Description

https://magento2.atlassian.net/browse/MAGECLOUD-4649

### Manual testing scenarios

1. Install magento 2.3.1 on cloud
2. Check that patch `MDVA-22979__fix_pagebuilder_module_2.3.1_v1.patch` was applied (cloud.log and changes from patch in files)
3. Install magento 2.3.2 on cloud
4. Check that patch `MDVA-22979__fix_pagebuilder_module_2.3.2_v1.patch` was applied (cloud.log and changes from patch in files)

### Contribution checklist
